frontend/src/types.ts

```diff
@@ -1940,6 +1940,15 @@ export interface PluginErrorType {
     event?: Record<string, any>
 }
 
+// The general log entry format that eventually everything should match
+export type LogEntry = {
```

Comment:
I'm going to follow up and try and unify these types as they are so close but needlessly different between the loggers